README.md

```diff
@@ -59,10 +59,26 @@ What we are generally NOT looking for includes:
 * Anything that violates the [Rust Community Code of Conduct](https://www.rust-lang.org/policies/code-of-conduct)
 * Rants or anything degrading to any part or member of the Community. Rather than submitting an article about what is wrong with something, we would much rather you write something that explains how you'd make it better.
 * Duplicates of recent posts (even with the wording changed slightly)
-* Links to crates or GitHub repos without some sort of context. We would much rather you submit a blog post introducing your project and how a Rust user might use it (and what it would help them do) and/or what you learned about Rust in the process of writing the project.
 * Anything behind a paywall (this includes Medium's paid article / members-only mechanism)
 * Anything that requires information to be shared/captured (like an email address) in order to access
 
+**Projects/Tooling Updates**
+
+There are further guidelines for the Projects/Tooling Updates Section
+
+We include:
+
+* Updates on tooling when in long form or framed as a tutorial (this can be through a blog, through GitHub, through a newsletter, and more) - it must have a high amount of Rust specific info (examples in Rust, notes on things learned about Rust in the process of creating/updating the project, etc.)
+* Updates on tooling that call out specific contributors - it is wonderful to highlight all the great people contributing to Rust OSS projects
```

Comment:
> Updates on tooling that call out specific contributors - it is wonderful to highlight all the great people contributing to Rust OSS projects

Can someone use attribution as a "pass" for not including Rust-specific info?

Reply:
Er, no. I will clarify.